doc/user/USAGE.md

```diff
@@ -1,63 +1,130 @@
-* **.ex2/.e/.exo/.g** : Exodus 2 file format
-* **.gml** : CityGML file format
-* **.pts** : Point Cloud file format
-* **.step/.stp** : CAD STEP exchange ISO format
-* **.iges/.igs** : CAD Initial Graphics Exchange Specification format
-* **.brep** : Open CASCADE BRep format
-* **.abc** : Alembic format
-* **.vdb** : VDB format (experimental)
-* **.obj** : Wavefront OBJ file format (full scene and default scene)
-* **.gltf/.glb** : GL Transmission Format (full scene and default scene)
-* **.3ds** : Autodesk 3D Studio file format (full scene)
-* **.wrl** : VRML file format (full scene)
-* **.fbx** : Autodesk Filmbox (full scene)
-* **.dae** : COLLADA (full scene)
-* **.off** : Object File Format (full scene)
-* **.dxf** : Drawing Exchange Format (full scene)
-* **.x** : DirectX Format (full scene)
-* **.3mf** : 3D Manufacturing Format (full scene)
-* **.usd** : Universal Scene Description (full scene)
-
-## Scene construction
-
-The **full scene** formats (.gltf/.glb, .3ds, .wrl, .obj, .fbx, .dae, .off, .x, .3mf, .usd) contain not only *geometry*,
-but also some scene information like *lights*, *cameras*, *actors* in the scene, as well as *texture* properties.
-By default, all this information will be loaded from the file and displayed. Use the `--geometry-only` [options](OPTIONS.md)
-to modify this behavior. For file formats that do not support it, **a default scene** is created.
-
-## Animations
-
-F3D can play animations for a number of file formats (.ex2/.e/.exo/.g, .gltf/.glb, .fbx, .dae, .x, .usd) if the file contains an animation.
-It is possible to select the animation to play using `--animation-index`, or to play all animations at once using `--animation-index=-1` (.gltf/.glb only).
-When F3D plays an animation, it assumes the time unit is in seconds to show accurate speed of animation. Use `--animation-speed-factor` if
-an adjustment is needed. By default, F3D will try update the scene 60 times per seconds, use `--animation-frame-rate` to change that if needed. Press "W" hotkey to cycle through available animations.
-
-## Plugins
-
-If you installed F3D using a package manager, it's possible that the packager chose to bundle the plugins in different packages or to list plugin dependencies as optional dependencies to reduce the reduce the number of dependencies of the main package.
-In this case, in order to open a file that requires a plugin, you will have to make sure all needed dependencies are installed and specify which plugin you want to load in order to be able to open this file. You can either use the `--load-plugins` option or add a line in the [configuration file](CONFIGURATION_FILE.md), if not already. Several plugins can be specified by giving a comma-separated list.
-Here is the list of plugins provided officially by F3D:
+F3D supports the following file formats:
+
+| Name | File Extension(s) | Type of Scene(s) Supported | Animations Supported |
+| -- | -- | -- | -- |
+| Legacy VTK | **.vtk** | Default | 
```

Comment:
I'd put either "Unsupported" or "Not supported by format" for the "Animations Supported" column.

Do you need help figuring out this information ?

Reply:
We have been discussed this in discord (do not hesitate to join btw: https://discord.f3d.app ) and creating a complete table may not be worth it tbh.